src/App.jsx

```diff
@@ -1,11 +1,211 @@
-
 import './App.css'
+import {useState} from "react";
+import Fruits from "./components/Fruits.jsx";
+import BigButtons from "./components/BigButtons.jsx";
+import Input from "./components/Input.jsx";
+import CheckRadioBtn from "./components/CheckRadioBtn.jsx";
 
 function App() {
+    const fruits = [" 🍓 Aardbeien"," 🍌 Bananen","🍏 Appels","🥝 Kiwi's"];
+
+    const [counterAardbeien, setCounterAardbeien] = useState(0);
+    const [counterBananen, setCounterBananen] = useState(0);
+    const [counterAppels, setCounterAppels] = useState(0);
+    const [counterKiwis, setCounterKiwis] = useState(0);
+
+    function resetAllCounters () {
+        setCounterAppels(0)
+        setCounterAardbeien(0)
+        setCounterKiwis(0)
+        setCounterBananen(0)
+    }
+
+    function handleSubmit (e) {
+        e.preventDefault();
+        console.log(
+            `${fruits[0]} ${counterAardbeien}, ${fruits[1]} ${counterBananen}, ${fruits[2]} ${counterAppels}, ${fruits[3]} ${counterKiwis}\n
+            ${formState.firstname}\n
+            ${formState.lastname}\n
+            ${formState.age}\n
+            ${formState.postal}\n
+            ${formState.frequency}\n
+            ${formState.timeframe}\n
+            ${formState.comments}\n
+            ${formState.conditions}`)
+    }
+
+    const [formState, setFormState] = useState({
+        firstname:'',
+        lastname:'',
+        age:0,
+        postal:'',
+        frequency:'iedere week',
+        timeframe:'Overdag',
+        comments:'',
+        conditions: false
+    })
+
+    function handleFormChange (f) {
+        const changedFirstName = f.target.name
+        const changedLastName = f.target.name
+        const changedAge = f.target.name
+        const changedPostal = f.target.name
+        const changedFrequency = f.target.name
+        const changedTimeframe = f.target.name
+        const changedComments = f.target.name
+        const valueConditions = f.target.type === "checkbox" ? f.target.checked : f.target.value
+        const changedConditions = f.target.name
+
+        setFormState({
+            ...formState,
+            [changedFirstName]: f.target.value,
+            [changedLastName]: f.target.value,
+            [changedAge]: f.target.value,
+            [changedPostal]: f.target.value,
+            [changedFrequency]: f.target.value,
+            [changedTimeframe]: f.target.value,
+            [changedComments]: f.target.value,
+            [changedConditions]: valueConditions
+        })
```

Comment:
Als de name zou corresponderen met de property naam van de formstate, dan kan dit een stuk simpeler
```suggestion
        setFormState({
            ...formState,
            [f.target.name]: f.target.type === "checkbox" ? f.target.checked : f.target.value,
        });
```